examples/Makefile.am

```diff
@@ -83,18 +82,20 @@ VPP_INPUT_CFLAGS = \
 
 if ENABLE_X11
 simpleplayer_LDADD = $(YAMI_DECODE_LIBS)
-simpleplayer_LDFLAGS = $(LIBYAMI_CFLAGS)
+simpleplayer_CPPFLAGS = $(YAMI_COMMON_CFLAGS)
```

Comment:
Missing `$(AM_CPPFLAGS)`.  `AM_` flags don't get automatically applied to per-target overrides, thus it must be added explicitly in the cases where the flags are overridden.